src/backends/beamcoder.ts

```diff
@@ -199,6 +208,17 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
         });
     }
 
+    set playbackRate(rate: number) {
```

Comment:
I think as a design philosophy, we wanter to keep `getFrameAtTime` relative to the real time of the video. The application can request scaled time values to `getFrameAtTime()`.

Ideally, something we can add on top of that is an interpolation FPS to achieive smoother results (when requested by the user through `interpolateFps`).

Reply:
Yeah, I've been playing with `interpolateFps,` but that didn't work as expected. It works on a video with a slow frame rate (which has all of the frames), but it didn't work when trying to interpolate 1 or 2 frame(s). We first need to generate those frames before we can smooth them out.